_plugins/research.rb

```diff
@@ -29,7 +29,7 @@ def generate(site)
             end
           end
         }
-        if fm.key? 'publications'
+        if fm.key? 'publications' && !fm['publications'].nil? && !fm['publications'].empty?
```

Comment:
This line fixes the broken build by checking for an empty/null value in the "publications" part of the front-matter before attempting to parse

Reply:
I'm confused why this was throwing an error. where was there no publications list?

Reply:
That said, it does seem to work now.